aten/src/ATen/native/quantized/TensorFactories.cpp

```diff
@@ -94,19 +94,39 @@ Tensor empty_per_channel_affine_quantized_other_backends_stub(
 
 // Create an empty quantized Tensor with size, based on the options
 // and quantization parameters of the input quantized Tensor
-Tensor empty_quantized(IntArrayRef size, const Tensor& qtensor) {
+Tensor empty_quantized(
```

Comment:
Previously this function used to get the tensor options from the qtensor. Now it seems like it can be overridden. What is the main motivation for having these extra options?

Also, can we update the comment above this function?

Reply:
it still has its previous functionality, but in order to copy to a new device, we have to be able to overwrite the device option. I added the other options for completeness. Without that it wouldn't be possible to make .to('cuda') or .to('cpu') work for quantized tensors